bot/exts/evergreen/tic_tac_toe.py

```diff
@@ -0,0 +1,318 @@
+import asyncio
+import random
+import typing as t
+
+import discord
+from discord.ext.commands import Cog, Context, check, group, guild_only
+
+from bot.bot import SeasonalBot
+from bot.constants import Emojis
+from bot.utils.pagination import LinePaginator
+
+CONFIRMATION_MESSAGE = (
+    "{opponent}, {requester} want to play Tic-Tac-Toe against you. React to this message with "
+    f"{Emojis.confirmation} to accept or with {Emojis.decline} to decline."
+)
+
+
+def check_win(board: t.Dict[int, str]) -> bool:
+    """Check from board, is any player won game."""
+    if (
+            # Horizontal
+            board[1] == board[2] == board[3]
+            or board[4] == board[5] == board[6]
+            or board[7] == board[8] == board[9]
+            # Vertical
+            or board[1] == board[4] == board[7]
+            or board[2] == board[5] == board[8]
+            or board[3] == board[6] == board[9]
+            # Diagonal
+            or board[1] == board[5] == board[9]
+            or board[3] == board[5] == board[7]
+    ):
```

Comment:
What do you think about the suggestion below to make the code cleaner to read?
```suggestion
    if any((
            # Horizontal
            board[1] == board[2] == board[3],
            board[4] == board[5] == board[6],
            board[7] == board[8] == board[9],
            # Vertical
            board[1] == board[4] == board[7],
            board[2] == board[5] == board[8],
            board[3] == board[6] == board[9],
            # Diagonal
            board[1] == board[5] == board[9],
            board[3] == board[5] == board[7]
    )):
```